src/lib/containers/GenericCard.tsx

```diff
@@ -284,27 +308,19 @@ export const SynapseCardLabel: React.FC<SynapseCardLabelProps> = props => {
         </>
       )
     }
-  } else {
-    return (
-      <>
-        {split.map((el, index) => {
-          const { baseURL, URLColumnName, wrapValueWithParens } = labelLink
-          const value = wrapValueWithParens ? `(${el})` : el
-          const href = `/${baseURL}?${URLColumnName}=${value}`
+  }
 
-          return (
-            <React.Fragment key={el}>
-              <a href={href} key={el} className={newClassName} style={style}>
-                {el}
-              </a>
-              {index < split.length - 1 && (
-                <span style={{ marginRight: 4 }}>, </span>
-              )}
-            </React.Fragment>
-          )
-        })}
-      </>
+  if (labelLink.tooltipText) {
```

Comment:
And this is the new code.  If tooltipText is set, then wrap the labelContent in a ReactTooltip.  Otherwise, just return the labelContent